lib/setup.py

```diff
@@ -98,19 +92,42 @@ def run(self):
 setuptools.setup(
     name=NAME,
     version=VERSION,
-    description=DESCRIPTION,
-    long_description=LONG_DESCRIPTION,
+    description="The fastest way to build data apps in Python",
+    long_description=(THIS_DIRECTORY / ".." / "README.md").read_text(),
+    long_description_content_type="text/markdown",
     url="https://streamlit.io",
     project_urls={
-        "Source": "https://github.com/streamlit/streamlit",
+        "Source Code": "https://github.com/streamlit/streamlit",
+        "Bug Tracker": "https://github.com/streamlit/streamlit/issues",
+        "Release notes": "https://docs.streamlit.io/library/changelog",
+        "Documentation": "https://docs.streamlit.io/",
+        "Community": "https://discuss.streamlit.io/",
```

Comment:
nit: we could add the Streamlit Twitter account here as well. Twitter is also one of the supported labels with an icon on pypi.

Reply:
Updated.